packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -76,14 +76,19 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     isLoading: isDashboardLoading,
   } = useCloudPulseDashboardByIdQuery(dashboardId);
 
+  const platformFilter =
+    dashboard?.service_type === 'dbaas' ? { platform: 'rdbms-default' } : {};
+
   const {
     data: resourceList,
     isLoading: isResourcesLoading,
   } = useResourcesQuery(
     Boolean(dashboard?.service_type),
     dashboard?.service_type,
     {},
-    {}
+    {
```

Comment:
we don't nee to spread here, just pass platformfilter

Reply:
ok, will hardcode